proto/celestia/qgb/v1/query.proto

```diff
@@ -31,6 +31,15 @@ service Query {
       returns (QueryLatestAttestationNonceResponse) {
     option (google.api.http).get = "/qgb/v1/attestations/nonce/latest";
   }
+  // LastValsetRequestBeforeNonce Queries last Valset request before nonce.
```

Comment:
What does `Valset` stand for? is it validator set? can we please add the full term in the description as well

Reply:
It's a type that describes validator sets:
https://github.com/celestiaorg/celestia-app/blob/4f9deb07b72e34f5eeeda87ef2a70f45f05c1c3f/proto/celestia/qgb/v1/types.proto#L17-L29